src/admin/user-groups/views/UserGroupOverviewPage.tsx

```diff
@@ -0,0 +1,41 @@
+import { UserGroupOverview } from '@meemoo/admin-core-ui';
+import React, { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
+import MetaTags from 'react-meta-tags';
+
+import { GENERATE_SITE_TITLE } from '../../../constants';
+import { withAdminCoreConfig } from '../../shared/hoc/with-admin-core-config';
+import { AdminLayout, AdminLayoutBody } from '../../shared/layouts';
+
+const UserGroupGroupOverviewPage: FunctionComponent = () => {
+	const [t] = useTranslation();
+
+	const renderPageContent = () => {
+		return <UserGroupOverview />;
+	};
```

Comment:
What's the benefit of this function?

Reply:
This was just to get all components uniform. They all have a layout with <MetaTags> and a {renderPageContent()}
But in this case, it doesn't really add much.

Do you prefer uniformity or simplicity? :p